### PR TITLE
added conwomap_1 example

### PR DESCRIPTION
**Pull request by tristan-ucar**

### CODE REVIEW

**Comment by erogluorhan** [comment]
This looks great overall. Only one inline comment should be addressed.

File: Plots/Contours/NCL_conwomap_1.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+"""
+NCL_conwomap_1.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple line contour plot
+   - Drawing a line contour plot with multiple colors
+   - Increasing the thickness of contour lines
```
Comment (erogluorhan) [inline Plots/Contours/NCL_conwomap_1.py:7]: This concept is from NCL script but it seems that no thickness adjustment is done to the contour plot in here. Playing with the argument `linewidths` in `u.plot.contour()` at Line 46 would be good to satisfy this concept.

File: Plots/Contours/NCL_conwomap_1.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/conwomap_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/conwomap_1_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
```
Comment (erogluorhan) [inline Plots/Contours/NCL_conwomap_1.py:18]: Probably no need for Cartopy in this example as all the projections could be simply removed (Please see my following comments for this).

File: Plots/Contours/NCL_conwomap_1.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+u = ds.u.isel(time=4)
+
+###############################################################################
+# Plot:
+
+#create figure
+plt.figure(figsize=(10,10))
+
+#create axes
+ax = plt.axes(projection=ccrs.PlateCarree())
```
Comment (erogluorhan) [inline Plots/Contours/NCL_conwomap_1.py:39]: Indeed no need for setting `project` as a Cartopy projection here for this example. Tried and saw no difference.

File: Plots/Contours/NCL_conwomap_1.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ax.set_aspect(1.5)
+
+#import colormap
+newcmp = gvcmaps.NCV_jet
+
+#contour plot data
+p = u.plot.contour(ax=ax,vmin=-1, vmax=10, transform=ccrs.PlateCarree(), levels=12, cmap=newcmp, add_labels=False)
```
Comment (erogluorhan) [inline Plots/Contours/NCL_conwomap_1.py:46]: Indeed no need for setting `transform` as a Cartopy projection here for this example. Tried and saw no difference.

File: Plots/Contours/NCL_conwomap_1.py (hunk 1 above)
Comment (tristan-ucar) [reply Plots/Contours/NCL_conwomap_1.py:7]: Good catch, made use of `linewidths` in the call to `contour`.

File: Plots/Contours/NCL_conwomap_1.py (hunk 2 above)
Comment (tristan-ucar) [reply Plots/Contours/NCL_conwomap_1.py]: Removed cartopy in latest commit.

File: Plots/Contours/NCL_conwomap_1.py (hunk 3 above)
Comment (tristan-ucar) [reply Plots/Contours/NCL_conwomap_1.py]: See above

File: Plots/Contours/NCL_conwomap_1.py (hunk 4 above)
Comment (tristan-ucar) [reply Plots/Contours/NCL_conwomap_1.py]: see above

**Comment by khallock** [comment]
Well done, @tristan-ucar! We really appreciate your effort on this!

**Pull request merged**